web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -489,6 +493,79 @@ describe("Formation - BusinessStep", () => {
         expect(screen.queryByText(Config.formation.fields.willPracticeLaw.label)).not.toBeInTheDocument();
       });
     }
+
+    describe("Business Designator Options based on Will Practice Law Answer", () => {
+      it.each(corporateLegalStructures)(
+        "Shows PA and PC options for Business Designator when Will You Practice Law is Yes",
```

Comment:
[sand] there are some describe statements that are capitalized and some aren't do we want to make them uniform?

Reply:
Maybe a broader conversation about code style with the team?